keyboards/xd68/rules.mk

```diff
@@ -32,4 +32,4 @@ FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
 UNICODE_ENABLE = no         # Unicode
 
-LAYOUTS = 65_ansi 65_iso
\ No newline at end of file
+LAYOUTS = 65_ansi 65_iso all
```

Comment:
```suggestion
LAYOUTS = 65_ansi 65_iso
```

Reply:
Thanks for that catch - I just read up on the docs for that var, and this shouldn't have been in there.